### PR TITLE
Release/1.4.3

### PR DESCRIPTION
**Pull request by sphuber**
Patch release containing two commits:

* Important fix to make SSL connections to RabbitMQ work
* Backport of bug fix in multiline CLI inputs that has already been merged into `develop`

### CODE REVIEW

**Comment by sphuber** [comment]
@ramirezfranciscof @chrisjsewell note that I also cherry-picked a commit  (commit 78cf6e1) that was already in `develop` but that is also needed in this patch release. This should be ready for review. Would be good if I could release this today.

**Comment by chrisjsewell** [comment]
I will have a look over lunch...

**Review by chrisjsewell — CHANGES_REQUESTED** [review]
minor nitpicks

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.4.3
```
[the remark this answers is not in the document]
Comment (sphuber) [reply CHANGELOG.md:3]: Fine with me. Shall we do this with the 1.5.0 release? We can then also add all the other retroactively.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+### Bug fixes
```
[the remark this answers is not in the document]
Comment (sphuber) [reply CHANGELOG.md:5]: Will do.

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisjsewell) [reply CHANGELOG.md:3]: Yep definitely also with 1.5.0, and there we can do some retroactive dating 👍

**Review by chrisjsewell — APPROVED** [review]

**Pull request merged**